Server/src/transport/plugin_hub.py

```
@@ -555,8 +589,89 @@ async def _handle_register_tools(self, websocket: WebSocket, payload: RegisterTo
+                    get_tools = getattr(cls._mcp, "get_tools", None)
+                    if not callable(get_tools):
+                        raise RuntimeError(
+                            "FastMCP exposes neither list_tools nor get_tools"
+                        )
+                    tools_by_name = await get_tools()
+                    published_tools = list(
+                        tools_by_name.values()
+                        if isinstance(tools_by_name, dict)
+                        else tools_by_name
+                    )
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)